### PR TITLE
ff7tk update and more

### PR DESCRIPTION
**Pull request by sithlord48**
Prep to use the upcoming ff7tk 0.83 
-  ff7tk has absorbed several files so we use the ff7tk versions now
- Show ff7tk version being used in about window
- Load ff7tk Translations
- Fixup Translation Search path to look in app path translations should fix out translation detection for deployments here
- Universal Mac Os Binary

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -34,13 +34,13 @@ jobs:
...
           package_suffix: 'linux64'
-          ff7tk_pack_suffix: "Linux_x86_64.tar.gz"
+          ff7tk_pack_suffix: "linux_x86_64.tar.gz"
           ff7tk_install_command_extract: "tar -xf"
           ff7tk_install_command_dest: "--one-top-level="
         - os: macos-latest
           package_extension: 'dmg'
-          package_suffix: 'macos64'
-          ff7tk_pack_suffix: "macos_x86_64.zip"
+          package_suffix: 'macos'
```
Comment (myst6re) [inline .github/workflows/build.yml:42]: Now I see this change, I wonder if it is OK to do it: Apple changed the main architecture of its processors 4 times now (powerpc, x86, x86_64 and currently a homemade ARM arch), if one plate-form needs to indicate the arch somewhere, that's will be Mac

File: src/Data.h [hunk 2]
```diff
@@ -49,7 +49,7 @@ class Data
 	static QMap<FF7Version, QString> ff7DataPathList();
 	static QString ff7KernelPath();
 	static QString charlgp_path();
-	static WindowBinFile windowBin;
+    static WindowBinFile windowBin;
```
Comment (myst6re) [inline src/Data.h:52]: Please use tabs

File: .github/workflows/build.yml (hunk 1 above)
Comment (sithlord48) [reply .github/workflows/build.yml:42]: the mac build has both x86_64 and arm compatibilty. there is no compatibilty for PPC or  x86(32-bit) in any verison of mac os this build will run on.

File: src/main.cpp [hunk 3]
```diff
@@ -67,6 +67,10 @@ int main(int argc, char *argv[])
 	} else {
 		Config::setValue("lang", QVariant());
 	}
+	QTranslator translator3;
+	if (translator2.load("ff7tk_" % lang, Config::programLanguagesDir()) || translator3.load("ff7tk_" % lang))
```
Comment (myst6re) [inline src/main.cpp:71]:
```suggestion
	if (translator2.load("ff7tk_" % lang, Config::programLanguagesDir()) || translator3.load("ff7tk_" % lang)) {
		app.installTranslator(&translator3);
	}
```

File: deploy/deployqt.cmake [hunk 4]
```diff
@@ -1,12 +1,11 @@
 if(APPLE)
     set(_target_file_dir "${_target_file_dir}/${_target_bundle_name}")
-
     set(DEPLOYQT_NAME mac)
     set(DEPLOYQT_ARGS)
 elseif(WIN32)
     set(DEPLOYQT_NAME win)
     list(JOIN LANGS "," QT_LANGS)
-    set(DEPLOYQT_ARGS --no-quick-import -network -xml -qml --translations ${QT_LANGS})
+    set(DEPLOYQT_ARGS "--no-quick-import -network -xml -qml --translations fr,jp")
```
Comment (myst6re) [inline deploy/deployqt.cmake:8]: Why this change?

**Comment by sithlord48** [comment]
Issues deploying on windows all of a sudden working on fixing it

**Comment by myst6re** [comment]
Thanks!
